pkg/payload/payload_test.go

```diff
@@ -123,3 +123,73 @@ func mustRead(path string) []byte {
 	}
 	return data
 }
+
+func TestExclude(t *testing.T) {
+	tests := []struct {
+		exclude     string
+		profile     string
+		annotations map[string]interface{}
+
+		isExcluded bool
+	}{
+		{
+			annotations: map[string]interface{}{},
+		},
+		{
+			exclude:     "",
+			annotations: map[string]interface{}{"exclude.release.openshift.io/identifier": "true"},
+		},
+		{
+			exclude:     "identifier",
+			annotations: map[string]interface{}{"exclude.release.openshift.io/identifier": "true"},
+			isExcluded:  true,
+		},
+		{
+			profile:     "",
+			annotations: map[string]interface{}{"include.release.openshift.io/edge": "true"},
+		},
+		{
+			profile:     "default",
+			annotations: map[string]interface{}{"include.release.openshift.io/edge": "true"},
+			isExcluded:  true,
+		},
+		{
+			profile:     "edge",
+			annotations: map[string]interface{}{"include.release.openshift.io/edge": "true"},
+		},
+		{
+			profile:     "",
+			annotations: map[string]interface{}{"include.release.openshift.io/default": "true"},
+		},
+		{
+			profile:     "default",
+			annotations: map[string]interface{}{"include.release.openshift.io/default": "true"},
+		},
+		{
+			profile:     "edge",
+			annotations: map[string]interface{}{"include.release.openshift.io/default": "true"},
+			isExcluded:  true,
+		},
+		{
+			profile:     "default",
+		},
+		{
+			profile:     "edge",
+		},
+	}
+	for _, tt := range tests {
+		ret := shouldExclude(tt.exclude, tt.profile, &lib.Manifest{
+			Obj: &unstructured.Unstructured{
+				Object: map[string]interface{}{
+					"metadata": map[string]interface{}{
+						"annotations": tt.annotations,
+					},
+				},
+			},
+		})
+		if ret != tt.isExcluded {
+			t.Errorf("(exclude: %v, profile: %v, annotations: %v) %v != %v", tt.exclude, tt.profile, tt.annotations, tt.isExcluded, ret)
+		}
+	}
+
```

Comment:
nit: useless empty space.  Can we drop this line?

Reply:
done :+1: